### PR TITLE
[Doc] Improving implicit-str-concat/details.rst

### PR DESCRIPTION
**Pull request by Lucas-C**
- [x] I think this PR can be labelled `skip-news`
- [x] Write comprehensive commit messages and/or a good description of what the PR does

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

Follow-up of #8648

I think it's worth adding this important warning.

Ping @Pierre-Sassoulas who reviewed & merged the previous PR

### CODE REVIEW

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Thank you ! Nice additional warning too ! I think we can also add https://github.com/pylint-dev/pylint/pull/8590 to related.rst.

**Comment by Lucas-C** [comment]
> Thank you ! Nice additional warning too ! I think we can also add #8590 to related.rst.

OK. I added a commit that initiates `related.rst`.

File: doc/data/messages/i/implicit-str-concat/related.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+* [PR #8590](https://github.com/pylint-dev/pylint/pull/8590): allowing parenthesized implicitly concatenated strings, to be more compatible with `black`
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/implicit-str-concat/related.rst:1]:
```suggestion
- ` allowing parenthesized implicitly concatenated strings, to be more compatible with black <https://github.com/pylint-dev/pylint/pull/8590>`_
```

File: doc/data/messages/i/implicit-str-concat/related.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+- ` allowing parenthesized implicitly concatenated strings, to be more compatible with black <https://github.com/pylint-dev/pylint/pull/8590>`_
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/implicit-str-concat/related.rst:1]:
```suggestion
- ` Allowing parenthesized implicitly concatenated strings, to be more compatible with black <https://github.com/pylint-dev/pylint/pull/8590>`_
```

File: doc/data/messages/i/implicit-str-concat/related.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+- ` Allowing parenthesized implicitly concatenated strings, to be more compatible with black <https://github.com/pylint-dev/pylint/pull/8590>`_
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/implicit-str-concat/related.rst:1]:
```suggestion
- `Allowing parenthesized implicitly concatenated strings, to be more compatible with black <https://github.com/pylint-dev/pylint/pull/8590>`_
```

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you again !

**Pull request merged**